backend/accounts.go

```diff
@@ -1072,10 +1073,23 @@ func (backend *Backend) createAndAddAccount(coin coinpkg.Coin, persistedConfig *
 
 	switch specificCoin := coin.(type) {
 	case *btc.Coin:
+		// Arguments have priority over config settings
+		gapLimits := backend.arguments.GapLimits()
+
+		if gapLimits == nil {
+			configReceive := uint16(backend.config.AppConfig().Backend.GapLimitReceive)
+			configChange := uint16(backend.config.AppConfig().Backend.GapLimitChange)
+			if configReceive > 0 && configChange > 0 {
+				gapLimits = &btctypes.GapLimits{
+					Receive: configReceive,
+					Change:  configChange,
+				}
+			}
+		}
```

Comment:
This function is already big enough so maybe it would be good to not add more code. I think we could create a `func (b *Backend) gapLimits()` that does all the same logic, WDYT? Then we can call it down on line 1078 similar to what we were doing before